Applications/SlicerApp/Testing/Python/SlicerMRBMultipleSaveRestoreTest.py

```diff
@@ -88,17 +88,15 @@ def test_SlicerMRBMultipleSaveRestore(self):
     import SampleData
     mrHeadVolume = SampleData.downloadSample("MRHead")
 
-    # Place a fiducial
-    markupsLogic = slicer.modules.markups.logic()
+    # Place a control point
+    pointListNode = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLMarkupsFiducialNode")
```

Comment:
This change is causing `py_SlicerMRBMultipleSaveRestoreTest` to fail because it uses the `GetNodeTagName` and not the `GetDefaultNodeNamePrefix` so the added node name is "MarkupsFiducial" and not "F", but it is trying to get the node by doing `pointListNode = slicer.util.getNode('F')`